moped-editor/src/components/GridTable/Filters.js

```diff
@@ -99,47 +99,45 @@ const useStyles = makeStyles((theme) => ({
 
 /**
  * Filter Search Component aka Advanced Search
- * @param {Object} filters - The current filters from useAdvancedSearch hook
  * @param {Function} setFilters - Set the current filters from useAdvancedSearch hook
  * @param {Function} handleAdvancedSearchClose - Used to close the advanced search
  * @param {Object} filtersConfig - The configuration object for the filters
  * @param {Function} resetSimpleSearch - Function to reset the simple search
+ * @param {Object} searchParams - The URL search params
+ * @param {Function} setSearchParams - Function to set the URL search params
```

Comment:
Thanks for being diligent and keeping these up to date. I find them easy to miss.